addons/event/views/event_tag_views.xml

```diff
@@ -24,6 +24,9 @@
                             <h1><field nolabel="1" name="name" placeholder='e.g. "Age Category"'/></h1>
                         </div>
                         <group>
+                            <!-- Placeholders in the layout for overrides -->
+                            <group id='left_preface' invisible='1'/>
```

Comment:
Better use name than id, no need of specific behavior linked to IDs in html (and what is a preface ?)